sdk/src/loom2d/Display/Stage.ls

```diff
@@ -430,6 +431,11 @@ package loom2d.display
                     }
 
                     break;
+                case StageScaleMode.MANUAL:
+                    scaleX = scaledWidth;
+                    scaleY = scaledHeight;
+                    x = 0;
+                    y = 0;
```

Comment:
Our default size as defined in our config file is 1280x720, but we'd like to use a 4x3 resolution (1280x960) for 4x3 screens, as determined at run time. This _looks_ like it might almost work, except in a situation where the screen is not perfectly 4x3 - in which case, the stage size wouldn't be centered, (and setting x & y to 0 here would prevent any opportunity to fix that). It probably needs to use if-else clauses like the ones for LETTERBOX.